Packs/Gmail/TestPlaybooks/playbook-GmailTest.yml

```
@@ -1578,7 +1578,7 @@ tasks:
       script: '|||send-mail'
```

not sure if it's required but better to be on the safe side
```suggestion
      script: 'Gmail|||send-mail'
```